website/content/specification/computer-example.xml

```diff
@@ -64,11 +72,22 @@
                             <define-field name="architecture" min-occurs="1">
                                 <formal-name>CPU Architecture</formal-name>
                                 <description>The Instruction Set Architecture (ISA) of the processor, <code>x86</code>, <code>x86-64</code>, <code>arm</code>, or an alternative.</description>
+                                <constraint>
+                                    <allowed-values target="." allow-other="yes">
+                                        <enum value="x86">x86 32-bit</enum>
+                                        <enum value="x86-64">x86 64-bit</enum>
+                                        <enum value="arm">arm</enum>
+                                    </allowed-values>
+                                </constraint>
                             </define-field>
                             <define-field name="speed" min-occurs="1">
                                 <formal-name>CPU Speed</formal-name>
                                 <description>The clock speed of the CPU in megahertz or gigahertz.</description>
+                                <constraint>
+                                    <matches target="." regex="\d+(?:\.\d+)?(?:MHz|GHz)"></matches>
+                                </constraint>
```

Comment:
Minor, non-blocking: I like this but I thought I had made this originally a `positiveInteger` datatype and was planning on accompanying that with something like a `speed-unit`, but I can see that instructive to me but a coding/modeling smell (I actually have never formulated this "style" question in the Metaschema community). Just a heads up, no suggested change here, feel free to resolve.

Reply:
I setup this constraint based on the current model. We could always break this field apart into value and unit data elements. We should probably do this in a different PR.